js/views/pageNavVw.js

```diff
@@ -630,6 +630,8 @@ module.exports = baseVw.extend({
   },
 
   mouseenterServerSubmenuTrigger: function(e) {
+    this.serverSubmenu.css('right', this.$('.popMenu-navBarSubMenu-pageNav').outerWidth());
```

Comment:
I'll retract that comment. Float introduces some issues with the container blocking the clicks to close the menu when you mouse off of it.

I'd rather avoid changing the right position in JavaScript, but for now I don't see a clean way to do it without reworking how the animation and hiding the menus works.